core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorManagerService.js

```diff
@@ -47,6 +47,22 @@ oppia.factory('SimpleEditorManagerService', [
             x: 0
           }
         }
+      },
+      TextInput: {
+        CUSTOMIZATION_ARGS:  {
+          placeholder: {
+            value: 'replacethis'
```

Comment:
We should change this to "Type your answer here" otherwise it would look like:

![image](https://user-images.githubusercontent.com/5758256/30778976-89c36e82-a097-11e7-9ae1-5588f122544d.png)

Reply:
Done.